data/human/news.txt

```diff
@@ -1836,7 +1836,56 @@ news "tourist pa system"
 		word
 			`"`
 
+news "republic navy advisory system"
+	location
+		government "Republic"
+	name
+		word
+			"Republic Navy Advisory System"
+	message
+		word
+			`"`
+		word
+			"Please be advised: a travel advisory remains in effect for "
+		word
+			"Albatross"
+			"Bloodsea"
+			"Buccaneer Bay"
+			"Covert"
+			"Deadman's Cove"
+			"Freedom"
+			"Greenrock"
+			"Haven"
+			"New Tortuga"
+			"Smuggler's Den"
+			"Stormhold"
+			"Thule"
+			"Zenith"
+		word
+			"."
+			"Please notify Republic officials immediately if you have been detained by any ship not granted legal authority to do so by the Republic Navy."
```

Comment:
```suggestion
		word
			"."
		word
			""
			"Please notify Republic officials immediately if you have been detained by any ship not granted legal authority to do so by the Republic Navy."
```
Otherwise you get something like `...for ZenithPlease...`

Reply:
This will end up with the travel advisory adjoined to, for example, an unrelated systems test of the radiation alert. I expect this will get worse in the future as more messages are added. I think the better way of handling this is to refactor the travel advisory into its own `phrase` so it doesn't interfere with other messages.